src/xdisplay.cpp

```diff
@@ -188,120 +188,160 @@ namespace xpyt
+                       const py::dict& metadata,
+                       const py::object& transient,
+                       const py::object& display_id,
+                       bool update,
+                       bool raw)
     {
         auto& interp = xeus::get_interpreter();
 
-        if (!obj.is_none())
+        for (std::size_t i = 0; i < objs.size(); ++i)
         {
-            if (hasattr(obj, "_ipython_display_"))
+            py::object obj = objs[i];
+            if (!obj.is_none())
             {
-                obj.attr("_ipython_display_")();
-                return;
-            }
+                if (hasattr(obj, "_ipython_display_"))
+                {
+                    obj.attr("_ipython_display_")();
+                    return;
+                }
 
-            py::object pub_data;
-            py::object pub_metadata = py::dict();
-            if (raw)
-            {
-                pub_data = obj;
-            }
-            else
-            {
-                const py::tuple& repr = mime_bundle_repr(obj, include, exclude);
-                pub_data = repr[0];
-                pub_metadata = repr[1];
-            }
-            pub_metadata.attr("update")(metadata);
+                py::object pub_data;
+                py::object pub_metadata = py::dict();
+                if (raw)
+                {
+                    pub_data = obj;
+                }
+                else
+                {
+                    const py::tuple& repr = mime_bundle_repr(obj, include, exclude);
+                    pub_data = repr[0];
+                    pub_metadata = repr[1];
+                }
+                pub_metadata.attr("update")(metadata);
 
-            nl::json cpp_transient = transient.is_none() ? nl::json::object() : nl::json(transient);
+                nl::json cpp_transient = transient.is_none() ? nl::json::object() : nl::json(transient);
 
-            if (!display_id.is_none())
-            {
-                cpp_transient["display_id"] = display_id;
-            }
+                if (!display_id.is_none())
+                {
+                    cpp_transient["display_id"] = display_id;
+                }
 
-            if (update)
-            {
-                interp.update_display_data(pub_data, pub_metadata, std::move(cpp_transient));
-            }
-            else
-            {
-                interp.display_data(pub_data, pub_metadata, std::move(cpp_transient));
+                if (update)
+                {
+                    interp.update_display_data(pub_data, pub_metadata, std::move(cpp_transient));
+                }
+                else
+                {
+                    interp.display_data(pub_data, pub_metadata, std::move(cpp_transient));
+                }
             }
         }
     }
 
+    void xdisplay(py::args objs, py::kwargs kw)
+    {
+        // Same as xdisplay - passing update = true to xdisplay_impl
```

Comment:
You might want to remove this comment or fix it to `// Same as xupdate_display - passing update = false to xdisplay_impl`

Reply:
Done.